static/src/javascripts/projects/commercial/modules/article-body-adverts.ts

```diff
@@ -54,7 +55,8 @@ const insertAdAtPara = (
 		});
 };
 
-const enableNearbyFilteringFix = () => true;
+const enableNearbyFilteringFix = () =>
+	!isInVariantSynchronous(spacefinderOkrMegaTest, 'control');
```

Comment:
i.e. enable the fix for:

- users not in `SpacefinderOkrMegaTest` (currently everyone)
- users in the `variant` group of `SpacefinderOkrMegaTest`